index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Compiled and minified CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Trails!</title>
+  </head>
+  <body>
+    <nav>
+      <div class="nav-wrapper navbar-fixed">
+        <a href="#" class="brand-logo">Trail Finder</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Search</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="container" id="welcome">
+      <br /><br />
```

Comment:
Trey says that <br> are too old school, we should use padding/margin in the finished project

Reply:
`<br/>` was the tag i was comment